frontend/src/pages/MemeVault.js

```diff
@@ -0,0 +1,65 @@
+import React, { useState, useEffect } from 'react'
+import { useHistory } from 'react-router-dom'
+import styled from 'styled-components'
+import { Images } from '../components/Images' 
+import { Button } from '../components/Button'
+
+const Wrapper = styled.section`
+
+`
+
+const Text = styled.h3`
+font-size: 20px;
+color: #F5F3F5;
+`
+
+const ErrorMsg = styled.p`
+  padding-top: 25px;
+  font-weight: 700;
+  font-style: italic;
+  color: red;
+`
+
+export const MemeVault = ({ username }) => {
+  const history = useHistory()
+  const [error, setError] = useState()
+  const accessToken = localStorage.getItem('accessToken')
+
+  const handleLogout = () => {
+    localStorage.removeItem('accessToken')
+    history.push('/')
+  }
+
+  useEffect(() => {
+    const abortController = new AbortController()
```

Comment:
where did you find this abort controller? something mentioned during a live session or something you figured out yourself?